extension/js/common/platform/store.ts

```diff
@@ -239,6 +239,19 @@ export class Store {
     return fromSession && !ignoreSession ? fromSession : undefined;
   }
 
+  static getKeysCurrentlyInSession = async (acctEmail: string) => {
```

Comment:
I would have it return a boolean, eg `hasAnyPassPhraseCurrentlyInSession`. That will simplify this method as well as the usage, because it's used that way anyway. Less code is better :)

Reply:
yes but then I work with an array of keys that the method returned:
```TS
      const keysInSession = await Store.getKeysCurrentlyInSession(acctEmail);
      if (keysInSession.length) {
        await Promise.all(keysInSession.map(async k => await Store.passphraseSave('session', acctEmail, k.longid, undefined)));
      }
```

Reply:
Sorry, I overlooked that. Got it.